src/server/player_sao.cpp

```diff
@@ -325,6 +325,14 @@ std::string PlayerSAO::generateUpdatePhysicsOverrideCommand() const
 	writeU8(os, !phys.sneak);
 	writeU8(os, !phys.sneak_glitch);
 	writeU8(os, !phys.new_move);
+	// new physics overrids since 5.8.0
```

Comment:
```suggestion
	// new physics overrides since 5.8.0
```

The typo is still there.